geopy/geocoders/geonames.py

```diff
@@ -163,9 +164,12 @@ def reverse(
             only, the value set during the geocoder's initialization.
 
         :param str feature_code: A GeoNames feature code
+            .. versionadded:: 1.18.0
 
         :param str lang: language of returned ``name`` element (the pseudo
             language code 'local' will return it in local language)
+            Full list of supported languages ISO639-2 can be found here:
+            http://www.loc.gov/standards/iso639-2/php/English_list.php
```

Comment:
It's very unlikely that geonames does indeed support all countries from that list. I presume it should be `https://www.geonames.org/countries/` instead.